delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/package.scala

```diff
@@ -49,9 +49,11 @@ package object model {
   }
 
   /**
-    * The id for the default elasticsearch view
+    * The id, name, and description for the default elasticsearch view
     */
-  final val defaultViewId = nxv + "defaultElasticSearchIndex"
+  final val defaultViewId      = nxv + "defaultElasticSearchIndex"
+  final val defaultName        = "elasticSearchView"
```

Comment:
We need to discuss quickly  these values with the rest of the team

Reply:
To be more flexible, we should make this values configurable

Reply:
Added a `ViewDefaults` to the SDK that could be reused also for BlazegraphViews.

* Naming is not great.
* `ViewId` is not included at this point.

Reply:
The view id for default view must remain a constant, for some operations we need it to be fixed (ex: querying all default views at once)